docs/samples/hello_world.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Hello World
+---
+
+# Hello World sample
+
+--8<-- "hello_world/hello_world.md"
+
+```ts
+--8<-- "hello_world/hello_world.ts"
```

Comment:
Just for my knowledge, what's the `--8<--` syntax?

Should we hold off on including this until we fix how the imports render, to avoid confusing people with an example that won't actually work?

Reply:
It's from the python markdown snippets extension: https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#snippets-notation

(It's a little scissor cutting on a line, although it took me a while to see that)

Now that PR #1365 is in I think the import problem is solved.